index.js

```diff
@@ -384,7 +384,7 @@ function signApplicationAsync (opts) {
         })
           .then(function () {
             debuglog('Signing... ' + opts.app)
-            execFileAsync('codesign', args.concat('--entitlements', opts.entitlements, opts.app))
```

Comment:
Sorry, don't quite understand what do you mean.

> What happened then is that verification (which should be after this promise fulfills) starts at a same time!?

How it can be? It is called in handler `promise.then`, so, it is ok.